LICENSE.md

```diff
@@ -0,0 +1,355 @@
+Mozilla Public License Version 2.0
+==================================
```

Comment:
It's your call, but personally I think this is a poor choice. BSD3 is the standard license in the Haskell community and personally I would need pretty strong reasons to use something less permissive.

Reply:
BSD3 is the most popular choice on hackage but I have been choosing the MPL for my own projects.

https://twitter.com/bodil/status/898917695084785664
https://youtu.be/aHVdJ0QT5rM?t=1034
https://github.com/bodil/spectre-of-free-software/blame/master/script.org#L185-L195

Comparing github's license summary BSD3 -> MPL2.0;
```
Permissions
-----------
   Commercial use
   Modification
   Distribution
   Private use
++ Patent use

Limitations
-----------
   Liability
   Warranty
++ Trademark use

Conditions
----------
   License and copyright notice
++ Disclose source
++ Same license (file)
```